modules/katamari/src/main/ts/ephox/katamari/api/Singleton.ts

```diff
@@ -18,7 +19,7 @@ export interface Value<T> extends Singleton<T> {
   readonly on: (fn: (data: T) => void) => void;
 }
 
-const revocable = <T> (doRevoke: (data: T) => void): Singleton<T> => {
+const singleton = <T> (doRevoke: (data: T) => void): Singleton<T> => {
```

Comment:
Is there a reason this was changed? Its still specifically a function to build a Singleton that can "revoke" the thing inside it, we just sometimes use it with `Fun.noop` to create a more generic singleton.

Reply:
It's the common interface between all now and not specific to revocable singletons. We could make `doRevoke` optional or defaulted but it didn't seem neccessary.

Reply:
Singleton seems like the wrong name for this now... but I don't have a better name to offer. In hindsight I do wonder if `Singleton.value()` should have been called `Singleton.empty()` for clarity.

Reply:
I think `empty` could be just as confusing, as it sounds like it should never have a value? Anyways, for now lets just leave it as is. If someone wants to desperately rename it, then feel free to do so in another PR 🤷